From 2820f71ae7590ad84c0c53528c162ed1c4a9cb44 Mon Sep 17 00:00:00 2001
From: Ranjani Vaidyanathan <Ranjani.Vaidyanathan@nxp.com>
Date: Fri, 2 Nov 2018 16:04:19 -0500
Subject: [PATCH 5019/5242] MLK-20222-3 drivers:keyboard: Update SCFW API

commit  f20ad72e8f60b9bac2fe21a15b7dcf98a42e46f7 from
https://source.codeaurora.org/external/imx/linux-imx.git

Update SCFW API to the following commit:
"
    ("430d1e3646fbe75e339e18abf2330565eac906e0")
    Author: Chuck Cannon <chuck.cannon@nxp.com>
    Date:   Fri Nov 2 15:25:45 2018 -0500

    SCF-105: RN updates.
"

Signed-off-by: Ranjani Vaidyanathan <Ranjani.Vaidyanathan@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 drivers/input/keyboard/imx_sc_pwrkey.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/input/keyboard/imx_sc_pwrkey.c b/drivers/input/keyboard/imx_sc_pwrkey.c
index 10c0d06..c89ca93 100644
--- a/drivers/input/keyboard/imx_sc_pwrkey.c
+++ b/drivers/input/keyboard/imx_sc_pwrkey.c
@@ -63,7 +63,7 @@ static int imx_sc_pwrkey_notify(struct notifier_block *nb,
 static void imx_sc_check_for_events(struct work_struct *work)
 {
 	struct input_dev *input = pdata->input;
-	bool state;
+	sc_bool_t state;
 
 	sc_misc_get_button_status(pdata->ipcHandle, &state);
 	/*
-- 
1.7.9.5

